src/reflect/scala/reflect/api/Trees.scala

```diff
@@ -2165,7 +2165,8 @@ trait Trees { self: Universe =>
   /** 0-1 argument list new, based on a symbol.
    *  @group Factories
    */
-  @deprecated("use q\"new ${sym.toType}(..$args)\" instead", "2.10.1")
+  private[this] final val depstr_New = "use q\"new ${sym." + "toType}(..$args)\" instead"
+  @deprecated(depstr_New, "2.10.1")
```

Comment:
implicitNotFound gets a pass, but actually all ordinary annotations? I think macros supplying constants would be OK.